metadata-store-directory-common/src/main/java/org/apache/helix/msdcommon/constant/MetadataStoreRoutingConstants.java

```diff
@@ -37,5 +37,6 @@
   // Field name in JSON REST response of getting sharding keys.
   public static final String SHARDING_KEYS = "shardingKeys";
 
-
+  // The key for system properties that contains the hostname of an instance
+  public static final String HOSTNAME_SYSTEM_PROPERTY_KEY = "instance_hostname";
```

Comment:
Nit: rename to MSDS_SERVER_HOSTNAME_KEY?

And for the actual value, let's use "msds_hostname" so it doesn't conflict with some general "instance_hostname" key.

Reply:
Sure.